XamarinCommunityToolkit/AttachedProperties/ProgressBarAttachedProperties.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.AttachedProperties
+{
+	public static class ProgressBarAttachedProperties
+	{
+		const string animatedProgress = "AnimatedProgres";
+
+		public static BindableProperty AnimatedProgressProperty = BindableProperty.CreateAttached(animatedProgress, typeof(double), typeof(ProgressBar), 0.0d, propertyChanged: ProgressBarProgressChanged);
+
+		static void ProgressBarProgressChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			var progressBar = bindable as ProgressBar;
+
+			if (progressBar == null)
+				return;
+
+			progressBar.ProgressTo((double)newValue, Convert.ToUInt32(Math.Max(0, 500)), Easing.Linear);
```

Comment:
Looks odd :) ```Convert.ToUInt32(Math.Max(0, 500))```

Reply:
```Easing.Linear``` and ```500```

Should we make these values customizable?

Reply:
I was thinking of doing that too in the beginning... but for the fluent animation this linear one looks great.
I guess what we could do is make an issue/enhancement on the Toolkit if someone asks for it. 

I will convert to using normal 500.